Formula/termtunnel.rb

```diff
@@ -0,0 +1,20 @@
+class Termtunnel < Formula
+  desc "Cross-platform terminal tunnel tool"
+  homepage "https://github.com/beordle/termtunnel"
+  url "https://github.com/beordle/termtunnel/archive/refs/tags/version-1.7.2.tar.gz"
+  sha256 "cd4d7b02bbae8a992f839b3d699923b0af4425ca5b2af6207e7f130fd18f1c8b"
+  license "MIT"
+
+  depends_on "cmake" => :build
```

Comment:
also it should depend on `libuv`?
```suggestion
  depends_on "cmake" => :build
  depends_on "libuv"
```

Reply:
similar to the idea of ​​Redis, dependencies are self-contained because it needs to be compiled into a static binary. (not dynamic linking)